tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+        "outDir": "./lib",
+        "allowJs": true,
+        "target": "es5",
```

Comment:
Do we need to downlevel to ES5?

Reply:
My intent with this pull request is to keep everything stable. The current JS code is ES5, so we should stay there.
Eventually we can adjust this.

Reply:
Don't get me wrong, I think we should change this, but the current minimum node version defined for this package is 6.x, so there are some discussions to be had before we can make this a es2030 ;)

Reply:
The code, at least in the master branch, uses ES6 features since https://github.com/mtth/avsc/pull/422 (classes, let/const, probably more).

Reply:
You're right. I also just  checked https://node.green/ - even on 6.x (I doubt anyone would still be using it and I don't think we should support anything smaller than 18.x) we can use most features.

@mtth - are you opposed to making this `es6` - it would mean some exports, etc. would change and JS code generated from the typescript would be slightly different.

Reply:
ES6 for `v6+` SGTM.